www/js/pages/real-account-opening/real-account-opening.controller.js

```diff
@@ -110,6 +110,10 @@
             vm.disableUpdatebutton = true;
             vm.error = {};
             let params = _.clone(vm.data);
+            const currency = appStateService.selectedCurrency || '';
+            if (currency) {
```

Comment:
@nazaninreihani here is better to use `!!` before `currency` in if condition since the value of `currency` is not as 'boolean`.

Reply:
@morteza-binary done here https://github.com/binary-com/mobile/pull/1171, thanks

Reply:
@morteza-binary btw I think it's redundant because if statement checks for boolean and the currency is either an empty string which is false or a currency, I'll close my PR since eslint doesn't allow to have !! in if statement either